bundles/binding/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/MyqData.java

```diff
@@ -82,16 +85,14 @@ public GarageDoorData getMyqData() {
 	 *            Attempt number when it recursively calls itself
 	 */
 	private String getGarageStatus(int attemps) {
-		if (this.sercurityTokin == null && Login()) {
+		if (sercurityTokin == null && !Login()) {
```

Comment:
I think you are correct in being cautious about the requests looking "abnormal" to chamberlain,  they seem to be very aggressive in fighting unauthorized 3rd parties from using their API,   A SmartThings engineering VP mentioned on their user forum that they are not pursuing an integration with them because they have sued their own partners (wink) over misuse of the api. If i had known this I would have told my garage guy to install another vendor.